core.lua

```diff
@@ -29,9 +33,33 @@ local recent_casts = {}
 
 -- UTILITY FUNCTIONS
 
+
+-- Function to populate voice options
+local function PopulateVoiceOptions()
+    local voices = C_VoiceChat.GetTtsVoices()
+    for _, voiceInfo in ipairs(voices) do
+        voiceOptions[voiceInfo.voiceID] = voiceInfo.name
+    end
+end
+
+
+local function TableHasValue(tbl, value)
+    for _, v in pairs(tbl) do
```

Comment:
These iterations can be avoided if the spell id is used as key instead of value.